content/changelog/2024/cody-promoted-prompts.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Promoted Prompts for VS Code and Cody Web"
+authors:
+  - name: Kelvin Yap
+    url: https://x.com/kelvinyap
+    avatar: https://storage.googleapis.com/sourcegraph-assets/blog/kelvin_avatar.png
+publishDate: 2024-10-30T17:34-07:00
+description: "Promoted Prompts elevates Prompts to Cody users to the top of the Prompts list or Prompt Library, allowing organizations to highlight specific Prompts that encourage best practices or are recommended to use."
```

Comment:
This is really hard to read...can you reword for clarity?

Something like:
"Prompts can now be promoted to show at the top of the Prompts list, allowing organizations to highlight..."